app/services/api/v1/data/historical_emissions_csv_content.rb

```diff
@@ -6,8 +6,8 @@ module Data
       class HistoricalEmissionsCsvContent
         def initialize(filter, output)
           @grouped_query = filter.call
-          @headers = filter.column_aliases
-          @headers.shift
+          # FIXME: To remove id and emissions: Should this be here?
```

Comment:
probably would be better to disable these columns for download in the configuration bit